tests/test.py

```diff
@@ -244,29 +244,29 @@ class TestPickingMethods(unittest.TestCase):
     #         len(components[0]) ,
     #     )
 
-    def test_compute_ICA_n_comp(self):
-        sample_read= Poly5Reader(sample_emg)
-        sample_emg_filtered = emg_bandpass_butter(sample_read, 1, 10)
-        sample_emg_filtered[1] = sample_emg_filtered[0]*1.5
-        sample_emg_filtered[2] = sample_emg_filtered[0]*1.7
-        doubled = np.vstack((sample_emg_filtered,sample_emg_filtered))
-        no_zeros = compute_ICA_n_comp(doubled, 1)
-        self.assertEqual(
-            (no_zeros.shape[0]),
-            6,
-        )
+    # def test_compute_ICA_n_comp(self):
```

Comment:
This test is commented out, but not added in another test file. It probably belongs in preprocessing?